### ISSUE TITLE
Imagemagick not working

### ISSUE THREAD

**Issue by maikeeb** [Zulko/moviepy#1197]
```
from moviepy.editor import *


def oneTransition(clipName, tranName):
    print(tranName)
    TextClip(clipName, color='white', size=(1920, 1080), bg_color='black', fontsize=50).set_duration(5).write_videofile(
        '../currentVideos/' + str(tranName).strip('\n') + '.mp4',
        fps=60,logger=None, remove_temp=True, threads=8)
```

### Expected Behavior

creates a 5-second clip with a black background with text in the middle saying a string

### Actual Behavior

sometimes works

sometimes gives me this error

```
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/home/maiku/.local/lib/python3.7/site-packages/moviepy/video/VideoClip.py", line 1161, in __init__
    subprocess_call(cmd, logger=None)
  File "/home/maiku/.local/lib/python3.7/site-packages/moviepy/tools.py", line 54, in subprocess_call
    raise IOError(err.decode('utf8'))
OSError: convert-im6.q16: attempt to perform an operation not allowed by the security policy `@/tmp/tmpmr4ard3d.txt' @ error/property.c/InterpretImageProperties/3666.
convert-im6.q16: no images defined `PNG32:/tmp/tmpupxr37mx.png' @ error/convert.c/ConvertImageCommand/3258.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/maiku/Desktop/offlinetvv montage maker/scripts/transition.py", line 6, in oneTransition
    TextClip(clipName, color='white', size=(1920, 1080), bg_color='black', fontsize=50).set_duration(5).write_videofile(
  File "/home/maiku/.local/lib/python3.7/site-packages/moviepy/video/VideoClip.py", line 1170, in __init__
    raise IOError(error)
OSError: MoviePy Error: creation of None failed because of the following error:

convert-im6.q16: attempt to perform an operation not allowed by the security policy `@/tmp/tmpmr4ard3d.txt' @ error/property.c/InterpretImageProperties/3666.
convert-im6.q16: no images defined `PNG32:/tmp/tmpupxr37mx.png' @ error/convert.c/ConvertImageCommand/3258.
.

.This error can be due to the fact that ImageMagick is not installed on your computer, or (for Windows users) that you didn't specify the path to the ImageMagick binary in file conf.py, or that the path you specified is incorrect


```

### Steps to Reproduce the Problem
I have no idea 

### Other

I am running this in threads with about 10 happening at the same time.

### Specifications

  - Python Version: 3.7.4
  - Moviepy Version: 1.0.0
  - Platform Name: ubuntu
  - Platform Version: 19.10

**Comment by tburrows13** [comment]
Hello, it looks like the fix is here: https://github.com/Zulko/moviepy/issues/401#issuecomment-278679961
You will need to find and edit your ImageMagick policy file.

**Comment by maikeeb** [comment]
I did that and it still has this error, where it sometimes doesn't work.

**Comment by maikeeb** [comment]
running this without threading doesn't have the same issue

**Comment by tburrows13** [comment]
That is very strange. Unfortunately I have no idea about a proper fix, the integration with ImageMagick not that stable. You can see in https://github.com/Zulko/moviepy/issues/1145#issuecomment-623594679 I've started a version of TextClip that doesn't use ImageMagick, which will probably be the default in v2.0.

**Comment by tburrows13** [comment]
See #1472 - TextClip's are being re-implemented in v2.0 so this will no longer be an issue.

**Comment by mathgenius1991712** [comment]
I am using Moviepy and TextClip on docker. And I have the same issue

**Comment by VinceDerPrince** [comment]
found a fix?

**Comment by gaojindev** [comment]
Yes, I managed to solve that
In dockerfile, I added the script that remove the original policy.yml file and add new policy.yml in which I removed that part

**Comment by prathapkr** [comment]
found any fix?

**Comment by gaojindev** [comment]
Hi, yes. I managed to find the solution
There is policy.xml file in imagemagic's installation directory where you need to remove one line 
That worked for me.

**Comment by sungeuns** [comment]
If you are using docker, you can use same solution [here](https://github.com/Zulko/moviepy/issues/401#issuecomment-278679961)

First, need to check that your ImageMagick `policy.xml` is working well in your local system.
And copy this file (normally exist in `/etc/ImageMagick-6/` but recommend copy in into project repo) into docker from Dockerfile.

```
COPY [LOCAL policy.xml PATH] /etc/ImageMagick-6/
```

**Comment by Y-Haneji** [comment]
> Hello, it looks like the fix is here: [#401 (comment)](https://github.com/Zulko/moviepy/issues/401#issuecomment-278679961) You will need to find and edit your ImageMagick policy file.

This fixed my similar issue. The policy had blocked the resource starting with "@".
